sympy/integrals/meijerint.py

```diff
@@ -2091,6 +2099,18 @@ def meijerint_inversion(f, x, t):
         shift = Add(*exponentials)
         f = Mul(*newargs)
 
+    if x not in f.free_symbols:
+        _debug('Expression consists of constant and exp shift:', f, shift)
+        from sympy import InverseLaplaceTransform, Eq, im
+        cond = Eq(im(shift), 0)
+        if shift.is_real == False:  # cond == False, issue #14978
+            _debug('but shift is nonreal, cannot be a Laplace transform')
+            return None
+        res = f*DiracDelta(t + shift)
+        _debug('Result is a delta function, possibly conditional:', res, cond)
+        return Piecewise((res.subs(t, t_), cond),
+                         (InverseLaplaceTransform(f_.subs(t, t_), x, t_, None), True))
```

Comment:
I don't think this (nonexistent) transform would be useful. It seems to be possible to have a Piecewise with only one component.

Reply:
ok, I've done this.